src/util-logopenfile.c

```diff
@@ -713,15 +716,22 @@ LogFileCtx *LogFileEnsureExists(LogFileCtx *parent_ctx, int thread_id)
  */
 static bool LogFileNewThreadedCtx(LogFileCtx *parent_ctx, const char *log_path, const char *append, int thread_id)
 {
+    static bool eve_suffix_init = false;
```

Comment:
is this shared between threads as well? If so it should probably be an atomic too?

Reply:
Looks like it is.